src/openassetio-python/package/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -513,6 +513,270 @@ def __testRegisterError(self, fixture_name, errorCode):
         self.assertEqual(actual_error.message, expected_error.message)
 
 
+class Test_getWithRelationship(FixtureAugmentedTestCase):
+    """
+    Check plugin's implementation of
+    managerApi.ManagerInterface.Test_getWithRelationship.
+    """
+
+    def setUp(self):
+        self.a_reference_to_a_readable_entity_with_relations = self._manager.createEntityReference(
+            self.requireFixture(
```

Comment:
[question] I take it that the sheer volume of fixtures to cover the whole matrix is what you meant by being a bit long-winded?

Reply:
Yeah, and the fragility of string literals etc...